client/packages/requisitions/src/RnRForms/DetailView/SidePanel.tsx

```diff
@@ -53,6 +63,31 @@ export const SidePanel = ({ data }: { data: RnRFormFragment }) => {
             <PanelLabel>{t('label.created')}</PanelLabel>
             <PanelField>{localisedDate(data.createdDatetime)}</PanelField>
           </PanelRow>
+          <PanelRow>
```

Comment:
Can this be wrapped in its own grid and given more padding so it's not so close to the uneditable text... also do you think the two text boxes are too close to each other???